tests/Console/Command/test-project/.php_cs

```diff
@@ -0,0 +1,18 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->in([
+        __DIR__
+    ])
+    ->exclude([
```

Comment:
maybe put sth excluded too, to verify that we take all "in" and "out", not only "in"